pyaedt/modeler/Primitives.py

```diff
@@ -1237,6 +1238,83 @@ def create_point(self, position, name=None, color="(143 175 143)"):
         self._refresh_points()
         return self._create_object(name)
 
+    @pyaedt_function_handler()
+    def create_plane(
+        self,
+        name=None,
+        plane_base_x="0mm",
+        plane_base_y="0mm",
+        plane_base_z="0mm",
+        plane_normal_x="0mm",
+        plane_normal_y="0mm",
+        plane_normal_z="0mm",
+        color="(143 175 143)",
+    ):
+        """Create a plane.
+
+        Parameters
+        ----------
+        name : str, optional
+            Name of the plane. The default is ``None``, in which case the
+            default name is assigned.
+        plane_base_x : str
+            X coordinate of the plane base. The default value is ``"0mm"``.
+        plane_base_y : str
+            Y coordinate of the plane base. The default value is ``"0mm"``.
+        plane_base_z : str
+            Z coordinate of the plane base. The default value is ``"0mm"``.
+        plane_normal_x : str
+            X coordinate of the normal plane. The default value is ``"0mm"``.
+        plane_normal_y : str
+            Y coordinate of the normal plane. The default value is ``"0mm"``.
+        plane_normal_z : str
+            Z coordinate of the normal plane. The default value is ``"0mm"``.
+        color : str, optional
+            String exposing 3 int values such as "(value1 value2 value3)". Default value is ``"(143 175 143)"``.
```

Comment:
```suggestion
            String exposing the three integer values for the color of the plane. The
            default value is ``"(143 175 143)"``.
```
I don't think the "such as "(value1 value2 value3)" is needed as the default provides a clearer example on its own.